grid/recruitment/get-stage-changes/maps/__snapshots__/breezy-hr.test.ts.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`recruitment/get-stage-changes/breezy-hr GetStageChanges when specified candidate does exist performs correctly 1`] = `
+Ok {
+  "value": Object {
+    "stages": Array [
+      Object {
+        "createdAt": "2022-12-14T11:23:18.812Z",
+        "current": true,
+        "id": "applied",
+        "name": "Applied",
+      },
+      Object {
+        "createdAt": "2022-12-14T11:21:30.696Z",
+        "current": false,
+        "id": "feedback",
```

Comment:
It is strange that the ID of the stage is not unique in the collection.

Reply:
There are also unique identifiers, but since it's pulling it from `stream` endpoint, there are also other events and those ids are tied to events, not stages.

Reply:
I could add model `StageChange` and have id of a event and additional field `stageId` which does not have to be unque, what do you think?

Reply:
> I could add model StageChange and have id of a event and additional field stageId which does not have to be unique, what do you think?

I do not get it, can you elaborate it more.

The option here could be to filter feedback out, as it is not recruitment stage.

Reply:
the endpoint offers events and contain unique ids for those events, but stage ids can be duplicated in these events as I do not request for list of stages, but stage changes (candidate stream). I propose to change the current model `Stage` in profile to model `StageChange` to mimic more an event entity rather than stage entity. We can add model `Stage` for use case to list available stages.

Reply:
or maybe `description` does not have to be dropped as it can describe reason of stage change

Reply:
Now I understand. It is actually valid for the candidate to be in the same stage multiple times.

Reply:
OK lets make the profile changes you described `Stage` -> `StageChange` and `id` -> `stageId`

Reply:
@martinalbert lets merge this PR and do the Stage -> StageChange and id -> stageId change in new PR